src/Oceananigans.jl

```diff
@@ -114,6 +114,13 @@ using DocStringExtensions
 using FFTW
 
 function __init__()
+    if VERSION >= v"1.11.0"
+        @info "Warning: You are using Julia v1.11 or later!"
+        @info "         Oceananigans is currently tested on Julia v1.10."
+        @info "         If you find issues with Julia v1.11 or later,"
+        @info "         please report at https://github.com/CliMA/Oceananigans.jl/issues/new"

```

Comment:
```suggestion
        @info """Warning: You are using Julia v1.11 or later!"
                 Oceananigans is currently tested on Julia v1.10."
                 If you find issues with Julia v1.11 or later,"
                 please report at https://github.com/CliMA/Oceananigans.jl/issues/new"""

```

this should work (with fixed alignment)